src/embed/app.spec.ts

```diff
@@ -479,6 +479,36 @@ describe('App embed tests', () => {
         });
     });
 
+    test('Should add flipTooltipToContextMenuEnabled flag to the iframe src', async () => {
```

Comment:
If it exists in all the classes write in a common point, instead of writing it seperately for all files.

Reply:
can you please point me out on which file is common for these?

Reply:
Done